lib/discover.js

```diff
@@ -150,6 +150,7 @@ function Discover (options, callback) {
 			return;
 		}
 
+		self.emit("HelloReceived");
```

Comment:
Please make this `self.emit("helloReceived");` so that it matches the `helloEmitted` below.